src/pages/Login/index.js

```diff
@@ -49,11 +49,12 @@ const Login = () => {
   };
 
   return (
+    <div className="loginDiv">
```

Comment:
non footer related changes would be better in another branch

Reply:
This is already in feature/login-page branch pull request, but as that hasn't been pulled across yet, that might be why it is appearing here as new.